File: number_theory/fibonacci/java/Fibonacci.java
Public class Fibonacci {
	public static voin main(String[] args{
		
	}

	public int getFibonacciAtIndex(int n) {
		if(n == 0) {
			return 0;
		} else if(n == 1) {
			return 1;
		} else {
			return getFibonacciAtIndex(n-1) + getFibonacciAtIndex(n-2);
		}
	}
}

File: number_theory/fibonacci/ruby/fibonacci.rb

# Returns a list of n fibonacci numbers
def fibonacci(n)
    fib_numbers = [0, 1]
    (2..n-1).each do |i|
        fib_numbers[i] = fib_numbers[i-1] + fib_numbers[i-2]
    end
    fib_numbers
end

puts fibonacci 10
    
